build-tools/packages/build-cli/src/library/commands/buildProjectPackageCommand.ts

```diff
@@ -0,0 +1,269 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	type IPackage,
+	type PackageFilterOptions,
+	type PackageSelectionCriteria,
+	type ReleaseGroupName,
+	selectAndFilterPackages,
+} from "@fluid-tools/build-infrastructure";
+import { type Command, Flags, ux } from "@oclif/core";
+import async from "async";
+
+import { BaseCommandWithBuildProject } from "./base.js";
+
+/**
+ * A re-usable CLI flag to parse release group names.
+ * Defined locally to avoid circular dependency with flags.ts.
+ */
+const releaseGroupNameFlag = Flags.custom<ReleaseGroupName>({
+	description: "The name of a release group.",
+	char: "g",
+	parse: async (input) => {
+		return input as ReleaseGroupName;
+	},
+});
```

Comment:
There should be a better solution for this - we should avoid redeclaring things and re-exporting stuff.